sdk/core/core-rest-pipeline/test/throttlingRetryPolicy.spec.ts

```diff
@@ -21,42 +21,57 @@ describe("throttlingRetryPolicy", function () {
     sinon.restore();
   });
 
-  it("It should retry after a given number of seconds on a response with status code 429", async () => {
-    const request = createPipelineRequest({
-      url: "https://bing.com",
+  const defaultDuration = 10 * 1000; // 10000 milliseconds
```

Comment:
![image](https://user-images.githubusercontent.com/10452642/157995282-1fadd15f-12cb-4c6e-b410-63b9d30eae67.png)
Hide whitespace to review test changes.